terraform/groups/ecs-service/main.tf

```diff
@@ -0,0 +1,82 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+terraform {
+  backend "s3" {
+  }
+  required_version = "~> 1.3"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+
+module "secrets" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.216"
```

Comment:
Is there a specific reason why we cannot use the latest version of `terraform-modules` `1.0.245`?

Reply:
oops did update this but must have not saved before push - apologies will catch it in the next push 👍